### ISSUE TITLE
Sphinx documentation?

### ISSUE THREAD

**Issue by cdeil** [cta-observatory/flexIRF#12]
How about adding Sphinx documentation?

It's very nice to have multiple linked pages, code examples, references, ...
It's already used by Gammalib / ctools / Astropy / Gammapy / open-astro-data-formats / ...

There's even a nice 1 hour tutorial how to get started:
https://github.com/cdeil/sphinx-tutorial

:-)

I'd be happy to make a pull request if you want it.

**Comment by TarekHC** [comment]
Yes, it should be in our roadmap. Indeed, all decent software projects nowadays use sphinx or something similar.

If you can implement the basics, then I could focus my efforts on documenting most classes/macros in "the proper way". That would be fantastic! :)
